index.html

```diff
@@ -5,6 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="./css/style.css" />
+    <script src="./javascript/index.js" defer></script>
```

Comment:
The Monk inside me wants to place the `script` either above or below both `link` elements.